src/WebJobs.Script/Workers/FunctionInvocationDispatcherState.cs

```diff
@@ -20,6 +20,16 @@ public enum FunctionInvocationDispatcherState
         /// invocations. All functions have been indexed. Listeners may not yet
         /// be not yet running.
         /// </summary>
-        Initialized
+        Initialized,
+
+        /// <summary>
+        /// The FunctionDispatcherState was fully initialized but has enconutered an error and is restarting
```

Comment:
Nit: Typo (please fix existing typos as well :)) FunctionDispatcher instead of FunctionDispatcherState . Also, change the comments to indicate, functions indexed, worker process encountered and error and restarting